comparisons/comparison-bpmn.md

```diff
@@ -309,14 +309,30 @@ functions:
 id: subflowloop
 name: SubFlow Repeat workflow
 version: '1.0'
-start: SubflowRepeat
+start: StartCount
 states:
+- name: StartCount
+  type: inject
+  data:
+    counter: 0
+  transition: SubflowRepeat
 - name: SubflowRepeat
-  type: subflow
-  workflowId: dosomethingandwaitformessage
-  repeat:
-    max: 10
-  end: true
+  type: operation
+  actions:
+    - subFlow: dosomethingandwaitformessage
+  actionDataFilter:
+    fromStateData: ${ .someInput }
+    toStateData: ${ .someInput }
+  transition: CheckCount
+- name: CheckCount
+  type: Switch
+  stateDataFilter:
+    input: ${ .counter += 1 }
```

Comment:
I think it would be clearer if we move this to the above operation state (in stateDataFilter output param)

Reply:
Done